jetstream/firefox-vpn-for-windows-10-users.toml

```
@@ -1,2 +1,592 @@
 [experiment]
-end_date = "2025-11-19"
\ No newline at end of file
+end_date = "2025-11-19"
+
+[metrics]
+
+# overwrite all irrelevant metrics with no-op
+
+# overwrite all metrics in Networking outcome: https://github.com/mozilla/metric-hub/blob/main/jetstream/outcomes/firefox_desktop/networking.toml
+[metrics.dns_lookup_time]
+
+
+[metrics.dns_lookup_time.statistics.deciles]
+select_expression = 'SUM(0)'
+data_source = "noop_source"
+
+[metrics.dns_lookup_time.statistics.kernel_density_estimate]
+select_expression = 'SUM(0)'
+data_source = "noop_source"
+
+[metrics.dns_lookup_time.statistics.empirical_cdf]
+select_expression = 'SUM(0)'
+data_source = "noop_source"
```

The `select_expression` and `data_source` parameters are on the metric, not on the statistic. You also don't need to worry about re-defining the statistics at all for the purpose of skipping metrics, so what you end up with should look like this:

```suggestion
[metrics.dns_lookup_time]
select_expression = 'SUM(0)'
data_source = "noop_source"
```